### ISSUE TITLE
False CCDs created, reading 204.9c - Ryzen 9 3900x

### ISSUE THREAD

**Issue by Alpha-087** [openhardwaremonitor/openhardwaremonitor#1277]
I'm going to go ahead and assume this is a bug of some sort with the software?

I noticed this started happening after updating my BIOS and chipset drivers.

My mobo is ASUS TUF GAMING X570-PLUS (WI-FI) - BIOS vers 1407
And my CPU is a Ryzen 9 3900x 
 
![HW](https://user-images.githubusercontent.com/65505880/82156004-b16c0300-9846-11ea-8057-b814d5aeac4a.png)

**Comment by Alpha-087** [comment]
Here is my attached report as well.  
[OpenHardwareMonitor.Report.txt](https://github.com/openhardwaremonitor/openhardwaremonitor/files/4640761/OpenHardwareMonitor.Report.txt)

**Comment by moellerm** [comment]
Looks to me like a collision with the hardware access. If another software tries to read the CPU package temperature at the same time, this will read 0x0FEF where it expects the values for the CCD temperatures. This value has the "valid" flag (0x800) set, and the temperature results in 204.875 °C ((0x0FEF & 0x7FF) * 0.125 - 49). The CCD sensors remain visible if one "valid" value has been seen there once (even if it currently does not provide valid values).

Does this happen as well if you do not run CoreTemp in parallel?

I can see if there are any accepted ways to synchronize with other software for this hardware access.

**Comment by Alpha-087** [comment]
Yes, this happened when I wasn't running Core Temp, though it had only generated one additional CCD that I noticed.  I only installed Core Temp after I noticed this happen, to see if it reported any similar issue.  Beyond that, I didn't have any other temp monitoring software running.

**Comment by moellerm** [comment]
I have added support for the mutex "Global\\Access_PCI" to interlock access to the PCI bus. This should avoid conflict with other tools that support this same mutex. I think Core Temp supports this mutex as well. You can test with this new version:

https://openhardwaremonitor.org/files/openhardwaremonitor-v0.9.3.4-alpha.zip

Please let me know the results.
